src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -42,7 +42,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- This prevents the default MsBuild targets from referencing System.Core.dll -->
     <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
-    <!-- These prevent the default MsBuild targets from referencing System.dll and mscorlib.dll -->
+    <!-- These prevent the default MsBuild targets from referencing System.dll and System.Private.CoreLib.dll -->
```

Comment:
Is `ExcludeMscorlibFacade` actually used anywhere? I am not able to find it in any target files.

Reply:
I looked too. Based on the comments, I was assuming it was in the official MsBuild targets. Let me do some more digging.

Reply:
I've looked under the dotnet organization as well as under microsoft/msbuild - no mentioned anywhere. I'll remove both properties.